tests/harvest/test_harvest_cli.py

```
@@ -4,31 +1,41 @@
-            argsline += f" --store {store_part}"
-
-        log.debug(f"testing equivlnt of python -m harvest {argsline}")
-        _main(*argsline.split(" "))  # pass as individual arguments
-        assert dump_path.exists(), "run did not create expected output"
-
-        # TODO consider some extra assertions on the result
+from sema.harvest.__main__ import _main, get_arg_parser
+
+
+@pytest.mark.parametrize(
+    "args_line, expected",
```

Ruff

Wrong type passed to first argument of `@pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)